### PR TITLE
Part of 936 implementation - intercept 555 error

### PR DESCRIPTION
**Pull request by rwrzesien**
This is part of implementation for https://github.com/peachworks/Peach-Web2/pull/57

### CODE REVIEW

**Comment by mbebas** [comment]
:+1:

File: src/providers/interceptor/interceptor.api.js [hunk 1]
```diff
@@ -64,7 +64,9 @@ class PeachInterceptorAPIProvider {
           }
 
           switch(rejection.status) {
-            case 401: {
+            case 401:
+            case 555:
```
Comment (rynomars) [inline src/providers/interceptor/interceptor.api.js]: Just curious, what is a 555 http status code?

**Comment by rynomars** [comment]
:+1:

File: src/providers/interceptor/interceptor.api.js (hunk 1 above)
Comment (jsmylnycky) [inline src/providers/interceptor/interceptor.api.js]: @rynomars Its a custom designated status code that the API uses to say that they're currently in maintenance mode.

Comment (rynomars) [inline src/providers/interceptor/interceptor.api.js]: @rwrzesien would you mind documenting line 68 so another developer does not see it as a mistake. Not knowing the intent of 555 could lead one to believe that the line is a mistake.

Comment (rwrzesien) [inline src/providers/interceptor/interceptor.api.js]: @rynomars Sure, np.

**Comment by jsmylnycky** [comment]
Merged master into this and added in the new peachjs.  It handles 555's properly since it doesnt have to deal with angular.

**Comment by jsmylnycky** [comment]
v2.0.5

**Pull request closed without merging**